### DISCUSSION TITLE
Circular Import Error

### DISCUSSION THREAD

**Discussion by RAMA-L7** [VaasuDevanS/cowsay-python#13] [Q&A]
Hello..
some error show in my device for ur code please help me.
ERROR: AttributeError: partially initialized module 'cowsay' has no attribute 'cow' (most likely due to a circular import)

**Comment by VaasuDevanS** [comment]
Can you try installing the latest version using pip (https://pypi.org/project/cowsay/4.0/)

**Reply by RAMA-L7** [reply]
Thak you sir for giving replay🙏
